youtube_dl/extractor/openload.py

```diff
@@ -14,7 +14,7 @@
 
 
 class OpenloadIE(InfoExtractor):
-    _VALID_URL = r'https://openload.(?:co|io)/(?:f|embed)/(?P<id>[a-zA-Z0-9-]+)'
+    _VALID_URL = r'https://openload.(?:co|io)/(?:f|embed)/(?P<id>[a-zA-Z0-9-\_]+)'
```

Comment:
_ is not a special character in Python's regular expressions. There's no need to escape it.